device-types/QNAP/TS-251-plus.yml

```diff
@@ -4,20 +4,23 @@ model: TS-251+
 slug: qnap-ts-251-plus
 part_number: TS-251+
 is_full_depth: false
-weight: 1.28
-weight_unit: kg
+subdevice_role: parent
 airflow: side-to-rear
 comments: '[TS-251+ | Hardware Specs | QNAP](https://www.qnap.com/en/product/ts-251+/specs/hardware)'
+weight: 1.28
+weight_unit: kg
+front_image: true
+rear_image: true
 power-ports:
   - name: Power
-    label: DC 12V
     type: dc-terminal
     maximum_draw: 19
     allocated_draw: 11
 interfaces:
   - name: eth0
-    label: '1'
     type: 1000base-t
   - name: eth1
-    label: '2'
     type: 1000base-t
+device-bays:
+  - name: Disk 1
+  - name: Disk 2
```

Comment:
These are not currently supported in the upstream NetBox. Please remove these definitions.